GOVERNANCE.md

```diff
@@ -0,0 +1,82 @@
+# Governance
+
+This project uses [Martha's Rules][marthas-rules] for making decisions.
+The first section below describes these rules;
+the second lists current voting members of the project and explains how to become one,
+while the third lists decisions made to date.
+
+## Making Decisions
+
+1.  Before each voting deadline (discussed below),
+    anyone who wishes may sponsor a proposal by filing an issue in the GitHub repository tagged "governance"
+    and either "propose-addition" or "propose-change".
+    Proposals must be filed at least 24 hours before a voting deadline in order to be considered,
+    and must include:
+    -   a one-line summary as the subject line of the issue
+    -   the full text of the proposal
+    -   any relevant background information
+    -   some discussion of pros and cons
+    -   some discussion of plausible alternatives (if any)
+
+2.  Once a person has sponsored a proposal, they are responsible for it.
+    The group may not discuss or vote on the issue
+    unless the sponsor is able to respond to questions and comments during the discussion period.
+
+4.  At or before the voting deadline,
+    a sense vote is cast for the proposal.
+    Each person may vote +1 (in favor), 0 (no opinion), or -1 (opposed).
+
+5.  If all votes are +1 or 0,
+    the proposal is accepted with no further discussion.
+
+6.  If the majority of votes are -1,
+    it is suspended pending further work by its sponsor
+    (who may instead choose to withdraw it).
+
+7.  If a minority of votes are -1,
+    members have one calendar week to add comments to the issue in its GitHub Issues thread.
+    At the end of this time a second vote is held
+    in which all votes must be either +1 or -1.
+    If a majority votes in favor the proposal is adopted.
+    Otherwise, the proposal is rejected.
```

Comment:
Additionally, I think it could be beneficial for the voting to be anonymous. That makes it easier for people to truly vote their opinion instead of adjusting for social consequences.

I also think there is value in not revealing in-progress vote counts to prevent people from just following the group without further thought.

Reply:
The setup to make these things "hidden" may be inconvenient though...

Reply:
voting in this case is public - if people object, they are going to need to explain their objections in order to improve the proposal. (think of e.g. parliament, where votes are a prelude or sequel to debate)